proto/ibc/core/channel/v1/channel.proto

```diff
@@ -74,6 +74,9 @@ enum State {
   // A channel has acknowledged the upgrade handshake step on the counterparty chain.
   // The counterparty chain that accepts the upgrade should set the channel state from OPEN to UPGRADETRY.
   STATE_TRYUPGRADE = 6 [(gogoproto.enumvalue_customname) = "TRYUPGRADE"];
+  // A channel has verified the counterparty chain is in UPGRADETRY.
+  // However, there are still in-flight packets on its end waiting to be flushed before it can move to OPEN.
```

Comment:
```suggestion
  // However, there are still in-flight packets on both ends waiting to be flushed before it can move to OPEN.
```

New protocol requires flushing on both ends to complete before we move to OPEN